diff --git a/dist/index.js b/dist/index.js
--- a/dist/index.js
+++ b/dist/index.js
@@ -2,6 +2,14 @@
-var __name = (target, value) => __defProp(target, "name", { value, configurable: true });
-var __export = (target, all) => {
-  for (var name in all)
-    __defProp(target, name, { get: all[name], enumerable: true });
-};
+var __name = (target, value) => {
+  if (target && typeof target === "object" || typeof target === "function") {
+    return __defProp(target, "name", { value, configurable: true });
+  }
+  return target;
+};
+var __export = (target, all) => {
+  if (!target || (typeof target !== "object" && typeof target !== "function")) {
+    return;
+  }
+  for (var name in all)
+    __defProp(target, name, { get: all[name], enumerable: true });
+};
